tests/modules/nf-core/mitohifi/findmitoreference/test.yml

```
@@ -0,0 +1,11 @@
+- name: mitohifi findmitoreference test_mitohifi_findmitoreference
+  command: nextflow run ./tests/modules/nf-core/mitohifi/findmitoreference -entry test_mitohifi_findmitoreference -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/mitohifi/findmitoreference/nextflow.config
+  tags:
+    - mitohifi/findmitoreference
+    - mitohifi
+  files:
+    - path: output/mitohifi/NC_072273.1.fasta
+      md5sum: 14c9906c7eccc0e57482112a452a031a
+    - path: output/mitohifi/NC_072273.1.gb
+      md5sum: 52d0342e6a9d0c3de076e68ef800eea9
```

The record got updated on Monday ! Here is the new MD5
```suggestion
      md5sum: 9063efc0ee83569566592f0198aa4eb5
```